web/firestore.rules

```diff
@@ -5,7 +5,22 @@ service cloud.firestore {
     //   - Authenticated users can add and edit messages.
     //   - Validation: Check name is same as auth token and text length below 300 char or that imageUrl is a URL.
     //   - Deletes are not allowed.
-    match /messages/{messageId} {
+    match /chatrooms/publicRoom/messages/{messageId} {
```

Comment:
We can combine public and private into the same block:
```suggestion
    match /chatrooms/{chatroom}/messages/{messageId} {
```

And remove the `privateRoom` below since it is the same